.ci/ci_cache_components.sh

```diff
@@ -121,7 +121,7 @@ create_cache_asset() {
 		echo $(basename "${path}") > "latest-${image_name}"
 		sudo cp "${path}" "${kata_dir}/osbuilder-${image_name}.yaml"  .
 	else
-		echo "${component_version}" >  "latest"
+		sudo echo "${component_version}" >  "latest"
```

Comment:
nominally I see what you are trying to do, but I suspect the `sudo` is only going to bind to the echo, and not the `>` redirect. You might need the idiom we have used elsewhere of `echo | sudo tee - a "latest"`. Let's see what the CI says.

Reply:
@grahamwhaley changes applied